framework/include/utils/InputParameters.h

```diff
@@ -996,6 +996,28 @@ InputParameters::setParamHelper<FunctionName, int>(const std::string & /*name*/,
   l_value = oss.str();
 }
 
+template<>
+inline
```

Comment:
In general I think that the framework has reached the size where we need to start removing inline code. If the compiler chooses to inline, that's fine but these methods are only called a few times during the setup so there is little reason to continue bloating the final executable size.